libs/ui/src/pure/Popover/index.tsx

```diff
@@ -56,7 +56,10 @@ export default function Popover(props: PopoverProps) {
           className={className}
           show={show}
           ref={setPopperElement as any}
-          style={styles.popper}
+          style={{
+            ...styles.popper,
+            zIndex: 999999,
```

Comment:
why hardcode the zIndex?

Wouldn't this. Do these help?
- https://github.com/floating-ui/floating-ui/issues/482
- https://stackoverflow.com/questions/52827235/z-index-on-popper-menu 

Or maybe you can find your way in the original documentation. Its very likely we are using it the wrong way, and this zIndex business is just covering up the issue (and potentially introducing new ones)

Reply:
Yup, I've gone through the original [documentation](https://floating-ui.com/docs/misc#z-index-stacking) and from this [issue](https://github.com/floating-ui/react-popper/issues/19#issuecomment-298627043) , I found out that `react-popper` doesn't really handle the z-index property on itself for the exact same reason as in the application itself there are so no,of places where z-index values are not consistent.

Reply:
And on the stackoverflow response does not work this usecase since we are specifically using `@reach/portal` lib to render popper element inside a portal.